examples/with-ecs-scheduling/main.tf

```diff
@@ -57,15 +57,38 @@ module "eventbridge" {
         attach_role_arn = true
 
         ecs_target = {
-          launch_type         = "FARGATE"
-          task_count          = 1
-          task_definition_arn = aws_ecs_task_definition.hello_world.arn
+          launch_type             = "FARGATE"
+          task_count              = 1
+          task_definition_arn     = aws_ecs_task_definition.hello_world.arn
+          enable_ecs_managed_tags = true
+          tags                    = { production = true }
 
           network_configuration = {
             assign_public_ip = true
             subnets          = data.aws_subnet_ids.default.ids
             security_groups  = [data.aws_security_group.default.arn]
           }
+
+          capacity_provider_strategy = [{
```

Comment:
Let's keep a value as a map, not lists of a map, when max one block is allowed. Please update the code in the examples and make sure that code works.

Reply:
Looks like `capacity_provider_strategy` allows multiple blocks. I updated the example a bit for ECS fargate type only since that's what it was originally.